src/modules/auth/index.js

```diff
@@ -173,8 +173,12 @@ export default class AuthModule extends Module {
 
 	async _updateUserInfo() {
 		let response;
+		// Obtain tenant for API call from URL params
+		const search = window.location.search;
+		const params = new URLSearchParams(search);
+		let activeTenant = params.get('tenant');
```

Comment:
`tenant` is optional ... if not present then it is not included in the userinfo call.

Reply:
@ateska Here it should not have any impact. If tenant is not in the url parameters, SeaCat Auth service is able to handle it (what I spoke of with Robin and from testing). This can mean that tenant in userinfo is not present or it can be also an empty list.

The API call looks like this

https://github.com/TeskaLabs/asab-webui/blob/8e4c8b8a05893897707b20adc761701572918ec6/src/modules/auth/api.js#L60

The `active_tenant` is send as a parameter in the request of the API call (not in the payload) and so when the `active_tenant` is undefined, null or empty string, the response is always without roles/resources/tenant in the userinfo.

But I might not see some things which will arise in the future. So I can separate the API call into 2 (one with `tenant` parameter in the API request and the other without it) and will use them accordingly, when active tenant from url params will be undefined.

Anyway, I did a bit of refactoring on tenant module to ensure that the application is loaded even when tenant is not in the userinfo response: https://github.com/TeskaLabs/asab-webui/pull/94/commits/8e4c8b8a05893897707b20adc761701572918ec6

Reply:
Well, what I mean mainly is that `userinfo(access_token, active_tenant)` signature is wrong since it doesn't make sense in deployments that don't use tenant.

Just move the code handling tenant inside `userinfo()` method, then is it encapsulated (invisible to an user programmer).

Reply:
Oki, moved and tested https://github.com/TeskaLabs/asab-webui/pull/94/commits/2e331b4772d803f9affdaef0e1af6df2681fc4a9